plugins/scaffolder/src/components/TaskPage/TaskErrors.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Box } from '@material-ui/core';
+import React from 'react';
+import { DismissableBanner } from '@backstage/core-components';
+import { TaskStream } from '../hooks/useEventStream';
+
+type TaskPageLinksProps = {
+  taskStream: TaskStream;
+};
+
+export const TaskErrors = ({ taskStream }: TaskPageLinksProps) => {
+  return taskStream.error ? (
```

Comment:
Does this really work? Seems to me that the taskStream itself doesn't necessarily have its reference (instance) value change? Since the error is what actually "drives" redraws here, it seems to me that if this works then it's accidentally because the surrounding component had to redraw itself, right? Essentially I'm asking if there should be a singe `error?: Error | undefined` prop instead

Reply:
It does work, I checked it on fresh instances. 

So I chose to take the entire TaskStream Prop as in the future we may be extracting errors from EventStream `stepLogs` , yeah for now we can only take a single prop, should I make that change?

Reply:
Made the change